lib/sequel_rails/railtie.rb

```diff
@@ -68,7 +68,7 @@ class Railtie < Rails::Railtie
     end
 
     initializer 'sequel.connect' do |app|
-      ::SequelRails.setup ::Rails.env unless app.config.sequel[:skip_connect]
+      ::SequelRails.setup ::Rails.env unless app.config.sequel[:skip_connect] or $0 =~ /rake/
```

Comment:
It might be good to turn this into a method at this point. The conditional is becoming obtuse.

How about extracting a `should_connect?` method with these two conditions. 

Railtie also has a `rake` hook where you could simply set `app.config.sequel[:skip_connect]` to true perhaps. I pored over ActiveRecord's railtie to figure out how they're avoiding this issue and couldn't find anything obvious to copy and reuse here.